thesis-de/content-de.tex

```diff
@@ -3158,43 +3198,34 @@ \section{Realisierung Grafana}
-beschränkt werden, welche User welche Einstellungen und Graphen sehen können.
-Dies wird in Grafana mit der Organisations-ID realisiert. Ein neu angelegter
-Benutzer wird dabei mindestens einer Organisation zugeordnet. Dies ist in
-diesem Projekt zum Beispiel für einen einzelnen Kunden von einer virtuellen
-Instanz vorgesehen. Durch die in
-Abschnitt~\ref{subsec:zusammenspiel_api_frontend} erläuterte Mapping-Tabelle
-wird sichergestellt, dass der Benutzer nur die zu Ihm zugeordneten Daten sehen
-kann. Administratoren oder andere Mitarbeiter eines Unternehmens, wie zum
-Beispiel ein Kundenberater, erhalten mehrere Zuordnungen von Organisations-IDs.
-Somit erhalten diese Zugriff auf die Dateninhalte des Kunden und können somit
-Ihre Aufgaben, wie die Beratung auf ein neues Produkt, oder Verbesserung der
-Gesamtperformance, durchführen. Andersherum muss für einen Kunden mit mehreren
-gemieteten virtuellen Instanzen kein neuer Benutzer angelegt werden,
-stattdessen wird seinem bestehenden Benutzer eine weitere Organisations-ID
-hinzugefügt.
-
-Ebenso ist es möglich, weitere Administratoren zu erstellen. Dies kann sowohl
-für lokale User als auch für LDAP\hyp{}User vergeben werden.  Sollte ein
-Unternehmen die LDAP\hyp{}Anbindung für die Authentifizierung an Grafana
-wünschen, so muss hier in dem Verzeichnis von der Grafana\hyp{}Installation die
-Datei grafana.ini angepasst werden. Alle Einstellungen, die hier vorgenommen
-werden, beziehen sich auf die Servereinstellungen, welche vom Kunden
-individuell angepasst werden kann.  Nachdem die Änderungen hier abgeschlossen
-sind, muss der Grafana\hyp{}Dienst neu gestartet werden. Dies wird mit
-folgendem Befehl durchgefüht:
+Nachstehend können noch diverse Feinkonfigurationen für einen Graphen
+spezifisch vorgenommen werden. Dieses Vorgehen wird nun für jeden Graphen
+erstellt, dessen Werte visualisiert werden sollen. Nachdem alle Graphen
+erstellt worden sind, muss das \gls{Dashboard} abgespeichert werden, damit die
+Konfigurationn nicht verloren gehen. Sollte das Speichern vom Administrator
+vergessen werden, so warnt Grafana bei dem Wechsel eines
+\glslink{Dashboard}{Dashboards}, das die Konfigurationen verloren gehen, falls
+diese nicht abgespeichert werden. Damit wie in dem Projekt gefordert eine
+\gls{API} Schnittstelle angesprochen werden kann, benötigt dies eine
+Plugininstallation. Es stehen viele verschiedene Plugins auf der Grafana
+Webseite zur Verfügung. Zu jedem Plugin gibt es hier eine kurze Erläuterung,
+sowie eine Schritt für Schritt Anleitung zur Installation dieses Plugins. Die
+Plugins werden hier über einen Kommandozeilenbefehl installiert.
+
+Die Grundkonfiguration sind nun abgeschlossen und Grafana kann verwendet
+werden. Um weiteren Benutzern Zugriff zu dem \gls{Dashboard} zu gewähren,
+können unter dem Reiter Admin > Global Users weitere Benutzer angelegt werden
+und Zugriff auf das Webinterface von Grafana erhalten. Insbesondere hier kann
```

Comment:
Tim hatte in irgendeinem kommentar gefragt warum du von deutsch auf englishc wechselst bei Webinterface